src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
+        int number = random.nextInt(100);
```

Comment:
```suggestion
        int number = random.nextInt(MAX_NUMBER);
```
avoid magic numbers ✨